oracle/oracles/pd.go

```diff
@@ -672,12 +672,6 @@ func (o *pdOracle) ValidateReadTS(ctx context.Context, readTS uint64, isStaleRea
 		return errors.Errorf("MaxInt64 <= readTS < MaxUint64, readTS=%v", readTS)
 	}
 
-	// only check stale reads and reads using `tidb_snapshot`
-	forTidbSnapshot := ctx.Value(ValidateReadTSForTidbSnapshot{}) != nil
```

Comment:
Shall we also remove the definition of `ValidateReadTSForTidbSnapshot`?

Reply:
I prefer not to remove it for now. We may need to check stale reads, e.g. after we add a switch to control checks for leader read.